uwblueprint/jump-math
# feat: Create Assessment Header

## Notion ticket link

[Add Create Assessment Header page](https://www.notion.so/uwblueprintexecs/47bf4727a2444bccad4a91952898b8f3?v=9658436bf9be409e90581631a7155f16&p=4245892b7ff44f09bea04e654ae779d6&pm=s)



## Implementation description
* Created Create Assessment Header and added it to the Create Assessment Page



## Steps to test
1. Go to http://localhost:3000/admin/assessments
2. Click on Create Assessment
3. The Create Assessment page should open (http://localhost:3000/admin/create-assessment)
4. The header is on the page, and date stamp is generated at the time of assessment creation (button click)
5. The header title should change from Untitled Assessment to whatever the user types into the first input field
6. The header matches the Figma design (https://www.figma.com/file/cRjJxm8087kkymfH4ieKau/Platform-Working-File?node-id=1095%3A65151&t=tH9IkeY0FV1Lf1Ei-4)



## What should reviewers focus on?
* 


## Checklist
